### ISSUE TITLE
IPv6

### ISSUE THREAD

**Issue by kevitra** [liquid8d/bbviewer#1]
With Windows 10 pro, if IPv6 is enabled the streaming does not work.  Disabling IPv6 immediately fixes the issue.

I personally had this issue and I've seen it reported on Reddit by other users.

**Comment by liquid8d** [comment]
Correct, this is a known issue (for 5.6). CBS has the same issue with their flashbacks, and BBViewer uses the same method to access streams as flashbacks on the CBS site. i reported it to them, so hopefully they correct it soon.
